### ISSUE TITLE
Class names issues

### ISSUE THREAD

**Issue by jpvaillancourt** [Weble/ZohoBooksApi#4]
Still have an uppercase issue on some classes that contains more than one words.

Class '\Webleit\ZohoBooksApi\Models\Customerpayments' not found. 

Fix on the Module.php:262
Inflector::singularize(Inflector::classify($className))

Honestly, I don't even know why you should do those minupulations since (new \ReflectionClass($this))->getShortName() is supposed to return the class name directly in the good format. Right?

And you also have an issue of the same kind on ProvidesModules line 20

**Comment by jpvaillancourt** [comment]
I have made several changes to make it work. The biggest issue was that you didn't had any information to create the cased version. You only had a lowered version of the string.

I'll make a pull request tomorrow, but here are the main modifications :

1 - I've transformed the $availableModules array into an associative array with the class names
2 - Created a method into the ProvidesModules class named getModuleShortClassName()
3 - Passing the ZohoBook instance into the constructor of the modules so they can have access to the method
4 - Fixed the getModelClassName() and the getPropertyList() method into the Module class
